assignments/lambda-classes.js

```diff
@@ -1 +1,162 @@
 // CODE here for your Lambda Classes
+
+class Person {
+	constructor(attr) {
+		this.name = attr.name;
+		this.age = attr.age;
+		this.location = attr.location;
+		this.gender = attr.genter;
+	}
+	speak() {
+		return `Hello, my name is ${this.name}, I am from ${this.location}`;
+	}
+}
+
+class Instructor extends Person {
+	constructor(attr) {
+		super(attr);
+		this.specialty = attr.specialty;
+		this.favLanguage = attr.favLanguage;
+		this.catchPhrase = attr.catchPhrase;
+	}
+	demo(subject) {
+		return `Today we are learning about ${subject}`;
+	}
+	grade(student, subject) {
+		`${student} receives a perfect score on ${subject}`;
+	}
+	gradeRandom(Student) {
+		let randomGrade = Math.floor(Math.random() * 100) + 1;
+		Student.grade = randomGrade;
+		if (Student.grade > 100) {
```

Comment:
I am not sure what's the logic here. you get grade > 100. your grade is 100. if it's less we do 0???